### PR TITLE
Adds React-App with XL template

### PR DESCRIPTION
**Pull request by GoldenCherro**

### CODE REVIEW

File: cherro-front/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...e/nicole/cherroapp/cherro-front/src/routes/app-link.jsx",[],"/home/nicole/cherroapp/cherro-front/src/common/error-boundary/error-boundary.jsx",[],"/home/nicole/cherroapp/cherro-front/src/helpers/params-helper.js",[],"/home/nicole/cherroapp/cherro-front/src/pages/home/home.jsx",[],"/home/nicole/cherroapp/cherro-front/src/pages/not-found/not-found.jsx",[],"/home/nicole/cherroapp/cherro-front/src/helpers/logger.js",[],"/home/nicole/cherroapp/cherro-front/src/hocs/with-layout/index.js",[],"/home/nicole/cherroapp/cherro-front/src/routes/scroll-to-top/index.js",[],"/home/nicole/cherroapp/cherro-front/src/pages/unexpected-error/index.js",[],"/home/nicole/cherroapp/cherro-front/src/config/constants.js",[],"/home/nicole/cherroapp/cherro-front/src/hocs/with-layout/layout.jsx",[],"/home/nicole/cherroapp/cherro-front/src/hocs/with-layout/with-layout.jsx",[],"/home/nicole/cherroapp/cherro-front/src/routes/scroll-to-top/scroll-to-top.jsx",[],"/home/nicole/cherroapp/cherro-front/src/pages/unexpected-error/unexpected-error.jsx",[],{"ruleId":"109","replacedBy":"110"},{"ruleId":"111","replacedBy":"112"},"no-native-reassign",["113"],"no-negated-in-lhs",["114"],"no-global-assign","no-unsafe-negation"]
```
Comment (mlvieras) [inline cherro-front/.eslintcache]: I don't think we should track this file.

File: cherro-front/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+    "jest": {
```
Comment (mlvieras) [inline cherro-front/package.json]: This isn't correctly indented, right?

File: cherro-front/.eslintcache (hunk 1 above)
Comment (mlvieras) [reply cherro-front/.eslintcache]: This file is still here. You need to explicitly delete it before adding it to the gitignore file 😄

Comment (GoldenCherro) [reply cherro-front/.eslintcache]: oopsi, okay!

**Review by mlvieras — APPROVED** [review]
🙌

**Pull request merged**